content/Why you should use GatsbyJS to build your landing page - Nicolás Santos.md

```diff
@@ -0,0 +1,26 @@
+![Gatsby](images/gatsby.png)
+
+# Why you should use Gatsby to build your landing page
+Having an awesome landing page is essential for your business.It allows your potential clients to know about what problem do you resolve for them and it's the visible face of your company. For that you need your page loads as fast as possible, all the major search engines such as Google, Bing, and Yahoo have it primary in search results and make available for everyone. And **Gatsby takes care of that and much more for you.**
```

Comment:
- There's a missing space between `business.` and `It`.
- `It allows your potential clients to know about what problem do you resolve for them` I would say `It allows your potential clients to know about what problems you solve for them`
- `For that you need your page loads as fast as possible,` Here it would be `to load as fast as possible,`
- `all the major search engines such as Google, Bing, and Yahoo have it primary in search results and make available for everyone.` This one seems kind of mixed up. Maybe `to appear in the first places of all the major search engines, such as Google, Bing, and Yahoo, and to be available to everyone.`